app/Http/Kernel.php

```diff
@@ -63,5 +63,6 @@ class Kernel extends HttpKernel
         'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
         'checkUser' => \App\Http\Middleware\CheckUser::class,
+        'jwt' => \App\Http\Middleware\JwtMiddleware::class,
```

Comment:
không nên sử dụng middleware này
vì bản thân của framework toàn là factory => tìm cách lợi dụng factory đó